clang/test/SemaCXX/attr-gnu.cpp

```diff
@@ -1,7 +1,7 @@
-// RUN: %clang_cc1 -std=gnu++17 -fsyntax-only -fms-compatibility -verify %s
-
-void f() {
-  // GNU-style attributes are prohibited in this position.
+// RUN: %clang_cc1 -std=gnu++17 -fsyntax-only -fms-compatibility -verify %s
```

Comment:
What... changed... here? I don't see any actual changes in the file...

Reply:
I'm pretty sure these are trailing whitespace removals.

Reply:
Huh, fair enough (though usually we want unrelated changes stripped from the patch, including trailing whitespaces).